configs/rule-sets/stylistic.js

```diff
@@ -229,6 +229,7 @@ export const stylisticRuleSet = {
         '@stylistic/type-named-tuple-spacing': 'error',
         '@stylistic/wrap-iife': ['error', 'inside'],
         '@stylistic/wrap-regex': 'off',
-        '@stylistic/yield-star-spacing': ['error', { before: false, after: true }]
+        '@stylistic/yield-star-spacing': ['error', { before: false, after: true }],
+        '@stylistic/jsx-pascal-case': 'off'
```

Comment:
Sure that we want to turn this off? Examples of incorrect code for this rule:

```jsx
<Test_component />

<TEST_COMPONENT />
```

Examples of correct code for this rule:

```jsx
<div />

<TestComponent />

<TestComponent>
  <div />
</TestComponent>

<CSSTransitionGroup />
```

Reply:
Yes. First of all this is JSX-specific so it should be disabled in the base config. For our `react-jsx` preset we have a similar rule from the react-plugin already enabled [here](https://github.com/enormora/eslint-config/blob/a0b2475530477fa0c126b306861ea554bbcf0550/configs/react-jsx.js#L30).

Reply:
Aha! 💡 Makes sense. I'm thinking about if this one makes the other one obsolete somehow / sometime 🤔